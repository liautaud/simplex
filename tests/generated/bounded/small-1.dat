4
8
2 2 4 4
-16 -2 14 -4 8 -4 -20 -36
-1 3 -3 -2
4 -3 1 -2
-1 4 -2 4
3 -3 1 -2
-4 4 -1 3
2 -4 4 -4
-1 3 -3 -3
-4 -4 -3 -2
