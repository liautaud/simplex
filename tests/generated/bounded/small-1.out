╒═══════════════════════════════════════╕
| Linear solver - OA course - ENSL 2017 |
╘═══════════════════════════════════════╛

The input linear program is:

Maximize  +2_x1  +2_x2  +4_x3  +4_x4  
Such that -1_x1  +3_x2  -3_x3  -2_x4  <= -16
          +4_x1  -3_x2  +1_x3  -2_x4  <= -2
          -1_x1  +4_x2  -2_x3  +4_x4  <= 14
          +3_x1  -3_x2  +1_x3  -2_x4  <= -4
          -4_x1  +4_x2  -1_x3  +3_x4  <= 8
          +2_x1  -4_x2  +4_x3  -4_x4  <= -4
          -1_x1  +3_x2  -3_x3  -3_x4  <= -20
          -4_x1  -4_x2  -3_x3  -2_x4  <= -36
          x_1, x_2, x_3, x_4 are non-negative

This linear program has a BOUNDED solution.
One optimal solution is: x_1 = 2, x_2 = 2, x_3 = 4, x_4 = 4
The value of the objective for this solution is: 40
The number of (second phase) pivots is: 1
The pivot rule used: random
